pyproject.toml

```diff
@@ -165,6 +165,7 @@ pybamm = [
 ]
 
 [tool.setuptools.packages.find]
+where = ["src"]
 include = ["pybamm", "pybamm.*"]
```

Comment:
We might be able to remove this now and rely on `setuptools` to find the package under `src`

```suggestion
```